sopel/bot.py

```diff
@@ -716,8 +716,11 @@ def dispatch(self, pretrigger):
                     if is_echo_message and not func.echo:
                         continue
 
+                    # Handle output prefix feature
+                    pfx = getattr(func, 'output_prefix', '')
```

Comment:
[nitpick] Not sure I like the `pfx` variable name.

Reply:
Fixed by eliminating the intermediate variable. 😝